proposals/nnnn-actors.md

```diff
@@ -608,25 +655,58 @@ In the absence of an explicitly-specified actor-isolation modifier (i.e, `noniso
 
 #### Protocol conformance
 
-When a given declaration (the "witness") satisfies a protocol requirement (the "requirement"), the actor isolation of the two declarations is compared. The protocol requirement can be satisfied by the witness if:
+When a given declaration (the "witness") satisfies a protocol requirement (the "requirement"), the protocol requirement can be satisfied by the witness if:
 
-* the witness and requirement have the same actor isolation,
-* the requirement is `async`, or
+* The requirement is `async`, or
 * the witness is non-isolated.
```

Comment:
Right yeah conformance makes sense this way 👍 

--- 

weird question; do would we want to allow `protocol X: Actor { func x() }` or is just just too weird?